src/config.js

```diff
@@ -32,6 +32,8 @@ export const urls = {
   thirdPartyFrameHost: env['thirdPartyFrameHost'] || 'ampproject.net',
   thirdPartyFrameRegex: thirdPartyFrameRegex || /^d-\d+\.ampproject\.net$/,
   cdn: env['cdnUrl'] || 'https://cdn.ampproject.org',
+  cdnProxyRegex: /^https:\/\/([a-zA-Z0-9_-]+\.)?cdn\.ampproject\.org/,
+  localhostProxyRegex: /^https?:\/\/localhost:[0-9]+\/(c|v)\//,
```

Comment:
What if the url is like `http://localhost.example.com`? I don't want to treat this as localhost